### DISCUSSION TITLE
Improve instructor experience for PL assignments that are manually hidden/shown to students

### DISCUSSION THREAD

**Discussion by firasm** [PrairieLearn/PrairieLearn#7390] [Ideas]
Hopefully this isn't an x-y problem, but because there currently isn't a way to hide questions that are being manually graded, I have set up my assessments to be hidden after students complete them (until I'm done manually grading them).
```
"showClosedAssessment": false,
"showClosedAssessmentScore": false,
```

Then I have to remember to set the flags below to `true` and if I forget, the scores and feedback aren't shown to students.

### Student view (assessments closed, scores not visible)
![Untitled](https://user-images.githubusercontent.com/2507459/229266317-91ca4c03-3d7e-4b83-a26e-f0c28564b632.png)

I know this is totally my responsibility to check and verify, but I wish there was an easy way to see which assessments were currently active, score was visible to students, and assessment open for viewing.

Something like this (note this is just a mockup)?

### Mockup of Instructor view
<img width="1277" alt="view" src="https://user-images.githubusercontent.com/2507459/229266654-b1ff3cc9-f295-49b6-af29-27f27a6a11f0.png">

Unfortunately, I can already see complications with this because of how access rules are constructed in PrairieLearn - we'd have to somehow look at "the most general" rule (maybe one that doesn't have UIDs listed?) that is "currently active based on the current time".

**Happy to flesh this out in a bit more detail if there is interest and appetite for something like this.**

**Comment by jonatanschroeder** [comment]
Like #6860?

**Reply by firasm** [reply]
Yes, exactly ! In this discussion I was trying to think of what the interface of this would look like.

I agree this is related to Manual Grading, but I think it would also be helpful for other assessment types as well (tests you don't want to release etc...).
